Index: mftp/application/api/request_processor/RequestMarshaller.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mftp/application/api/request_processor/RequestMarshaller.php	(revision fb65ea25d109aaf9f8afdf630d506342ba6d262b)
+++ mftp/application/api/request_processor/RequestMarshaller.php	(date 1554255925000)
@@ -35,12 +35,7 @@
         }
 
         private function applyConnectionRestrictions($connectionType, $configuration) {
-            $license = readDefaultMonstaLicense();
-            if (is_null($license) || !$license->isLicensed())
-                return $configuration;
-
             $applicationSettings = new ApplicationSettings(APPLICATION_SETTINGS_PATH);
-
             $connectionRestrictions = $applicationSettings->getUnblankedConnectionRestrictions();
 
             if (is_array($connectionRestrictions)) {
@@ -229,4 +224,4 @@
                 'data' => $encodedContents
             );
         }
-    }
\ No newline at end of file
+    }
Index: mftp/application/api/system/ApplicationSettings.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mftp/application/api/system/ApplicationSettings.php	(revision fb65ea25d109aaf9f8afdf630d506342ba6d262b)
+++ mftp/application/api/system/ApplicationSettings.php	(date 1554255893000)
@@ -345,14 +345,6 @@
             $restrictions = $this->getSetKey(self::$KEY_CONNECTION_RESTRICTIONS);
 
             if (is_array($restrictions)) {
-                $license = readDefaultMonstaLicense();
-                if (is_null($license) || !$license->isLicensed()) {
-                    if (array_key_exists("types", $restrictions))
-                        return array("types" => $restrictions["types"]);
-
-                    return $this->getDefaultValue(self::$KEY_CONNECTION_RESTRICTIONS);
-                }
-
                 $restrictions = $this->blankArray($restrictions, array("authenticationModeName", "initialDirectory"));
             }
 
@@ -631,4 +623,4 @@
         public function setSkipMacOsSpecialFiles($skipMacOsSpecialFiles) {
             $this->setBool(self::$KEY_SKIP_MAC_OS_SPECIAL_FILES, $skipMacOsSpecialFiles);
         }
-    }
\ No newline at end of file
+    }
